ci/build-push-arm64v8.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+if ! docker buildx ls | grep arm
+then
+    echo "Your Buildx seems to lack ARM architecture support"
```

Comment:
Hmm don't we need to check for particular flavor of ARM? i.e. `v7` `v8` etc ?

Reply:
I guess we could 🤷‍♂️